src/dealWizard/wizardManager.ts

```diff
@@ -0,0 +1,115 @@
+import { autoinject } from "aurelia-framework";
+import { PLATFORM } from "aurelia-pal";
+import { RouteConfig } from "aurelia-router";
+import { WizardService, IWizardState, IWizardStage } from "services/WizardService";
+import { DealRegistrationData, IDealRegistrationData } from "entities/DealRegistrationData";
+import { IStageMeta, WizardType, STAGE_ROUTE_PARAMETER } from "./dealWizardTypes";
+
+@autoinject
+export class WizardManager {
+  public wizardState: IWizardState<IDealRegistrationData>;
+
+  // a meta configuration passed to each stage component in the view
+  // for stage components to know which wizardManger they belong to and what wizardType it is
+  public stageMeta: IStageMeta;
+
+  // view of the currently active stage
+  public view: string;
+
+  // view model of the currently active stage
+  public viewModel: string;
```

Comment:
```suggestion
  public viewPath: string;

  // view model of the currently active stage
  public viewModelPath: string;
```

Self-explanatory? yay - nay?